man/describe_distribution.Rd

```diff
@@ -71,8 +67,6 @@ the first centrality index (which is typically the median).}
 \item{iterations}{The number of bootstrap replicates for computing confidence
 intervals. Only applies when \code{ci} is not \code{NULL}.}
 
-\item{threshold}{For \code{centrality = "trimmed"} (i.e. trimmed mean), indicates the fraction (0 to 0.5) of observations to be trimmed from each end of the vector before the mean is computed.}
```

Comment:
@IndrajeetPatil This workflow again introduced errors automatically. :-/

Reply:
Hmm, I don't get why this workflow misbehaves only with this file in the entire ecosystem. 
There must be something different about it.

Reply:
But, until we figure that out, we can delete the workflow file.